modules/multicast/multicast.c

```diff
@@ -309,6 +315,23 @@ static int cmd_mcreg(struct re_printf *pf, void *arg)
 		goto out;
 	}
 
+#ifdef HAVE_GETIFADDRS
+	if (str_isset(mccfg.iface)) {
+		unsigned int if_index;
+		if_index = if_nametoindex(mccfg.iface);
+		if (!if_index) {
+			warning("Can not find interface %s.\n", &mccfg.iface);
```

Comment:
"multicast: could not find interface %s.\n"

Reply:
Done.